lib/Renard/Curie/Data/PDF.pm

```diff
@@ -2,43 +2,101 @@ use Modern::Perl;
 package Renard::Curie::Data::PDF;
 
 use Capture::Tiny qw(capture_stdout);
+use XML::Simple;
+use Path::Tiny;
+use Alien::MuPDF 0.004;
```

Comment:
I added the Alien::MuPDF library. This library builds the mupdf library and places it into a special directory.